src/lib.rs

```diff
@@ -50,41 +50,140 @@ use std::{error, fmt};
 use std::str::FromStr;
 use std::fmt::{Display, Formatter};
 
+#[cfg(feature = "try_from")]
```

Comment:
What is the compatibility of `try_from`? Other changes were made to this library to push compatibility back to 1.14.

Reply:
AFAIK it's new in 1.25, that's why I put it behind a feature gate. You can use it if your rustc version is new enough, but you can ignore it otherwise. Unfortunately I couldn't find a way to express "only activate if rustc version >= 1.25.0", so the user has to activate it manually. But this reminds me of a TODO: run the stable test with the feature activated and add a test for `TryFrom`.

Reply:
#33417